scripts/demos/relay-1/docker/Dockerfile.dockerignore

```
@@ -0,0 +1,11 @@
+target/
```

Hadolint (2.15.1)

[error] unexpected 't'
expecting '#', ADD, ARG, CMD, COPY, ENTRYPOINT, ENV, EXPOSE, FROM, HEALTHCHECK, LABEL, MAINTAINER, ONBUILD, RUN, SHELL, STOPSIGNAL, USER, VOLUME, WORKDIR, a pragma, end of input, or whitespaces

(DL1000)